ros_buildfarm/release_job.py

```diff
@@ -663,6 +663,12 @@ def _get_binarydeb_job_config(
         'binarydeb/*.ddeb',
     ]
 
+    build_environment_variables = None
+    if build_file.build_environment_variables:
+        build_environment_variables = [
+            '%s=%s' % (var, value)
+            for var, value in build_file.build_environment_variables.items()]
```

Comment:
If this code block is changed to make `build_environment_variables` always a list (potentially empty) the "downstream code can be simplified:
* `@[if build_environment_variables]@` / `@[end if]@` could be removed from the template

Reply:
That's a great point! I'll do that.

Reply:
On second thought.. I'm not sure that will work. How does an empty list get handled by argparse's nargs="*" behavior. If we invoke commands `--env-vars` or `--env-vars --other-argument`. Will that be an empty list rather than None?

Reply:
```
import argparse
p = argparse.ArgumentParser()
p.add_argument('--foo', nargs='*')
p.parse_args(['--foo'])
```